src/generic/grid.cpp

```diff
@@ -2851,10 +2851,14 @@ void wxGrid::Init()
     m_gridLinesEnabled = true;
     m_gridLinesClipHorz =
     m_gridLinesClipVert = true;
-    m_cellHighlightColour = *wxBLACK;
+    m_cellHighlightColour = wxSystemSettings::GetColour(wxSYS_COLOUR_HIGHLIGHTTEXT);
     m_cellHighlightPenWidth = 2;
     m_cellHighlightROPenWidth = 1;
-    m_gridFrozenBorderColour = *wxBLACK;
+    if ( wxSystemSettings::GetAppearance().IsDark() )
+        m_gridFrozenBorderColour = *wxWHITE;
+    else
+        m_gridFrozenBorderColour = *wxBLACK;
+
```

Comment:
Is there any reason not to make this one the same as `m_cellHighlightColour` by default?

Reply:
I was trying to leave that part of the colors consistent with what was there before.   Previously, the currently selected cell has a black border while the rest of the selection has a bluish color (I'm color blind, so not really sure).  I originally had this as using the highlight color, but due to the appearance change I pulled this back.   My goal was to not have any visual change in "Light mode", but fix the main issues in dark mode.  I'd be OK with using highlight color.